src/DateRangePickerInput/web/DateRangePickerInput.js

```diff
@@ -377,8 +382,11 @@ DateRangePickerInput.propTypes = {
     from: PropTypes.bool,
     to: PropTypes.bool,
   }),
+  range: PropTypes.shape({
```

Comment:
also this prop can be renamed to `selectionRange` because it's too confusing otherwise